src/applications/disability-benefits/all-claims/content/disabilityLabels.js

```diff
@@ -0,0 +1,837 @@
+  6450: 'sleep disturbances',
+  6460: 'smell, loss of sense of',
+  6470: 'spine',
+  6480: 'spleen condition',
+  6490: 'sprains',
+  6500: 'Sternberg\'s sarcoma',
+  6510: 'stomach condition',
+  6520: 'stone(s)',
+  6530: 'strains',
+  6540: 'strep throat',
+  6550: 'stress',
+  6560: 'stress disorder',
+  6570: 'stroke',
+  6580: 'stump',
+  6590: 'surgery, residuals',
+  6600: 'synoviosarcoma',
+  6610: 'synovitis',
+  6620: 'syphilis',
+  6630: 'syringomyelia',
+  6640: 'systemic lupus',
+  6650: 'taste, loss of sense of',
+  6660: 'tendinitis',
+  6670: 'tendonitis',
+  6680: 'tenosynovitis',
+  6690: 'testicular condition, bilateral',
+  6700: 'testicular condition, left',
+  6710: 'testicular condition, right',
+  6720: 'thiamine deficiency',
+  6730: 'thigh condition',
+  6740: 'thoracic sprain',
+  6750: 'thoracic strain',
+  6760: 'throat condition',
+  6770: 'thromboangiitis, obliterans (Buerger\'s disease)',
+  6780: 'thrombocytopenia',
+  6790: 'thrombocytopenic purpura',
+  6800: 'thrombosis, brain',
+  6810: 'thumb',
+  6820: 'thyroid condition',
+  6830: 'thyroid nodular disease, non-malignant',
+  6840: 'tinea pedis',
+  6850: 'tinnitus',
+  6860: 'toe(s)',
+  6870: 'tongue condition',
+  6880: 'tooth condition',
+  6890: 'Tuberculosis',
+  6900: 'tumor(s) - malignant glomus',
+  6910: 'tumor(s) - malignant granular cell',
+  6920: 'tumors - brain',
+  6930: 'tumors - central nervous system',
+  6940: 'tumors - malignant',
+  6950: 'tumors - peripheral nerves',
+  6960: 'tumors - spinal cord',
+  6970: 'ulcer condition',
+  6980: 'Gulf War veteran with undiagnosed illness',
+  6990: 'Unemployability',
+  7000: 'uterus (disease, injury, or adhesions of)',
+  7010: 'uterus (displacement of)',
+  7020: 'uterus (prolapse)',
+  7030: 'uterus (removal of)',
+  7040: 'uveitis',
+  7050: 'vaginal condition',
+  7060: 'vaginitis',
+  7070: 'valvular heart disease',
+  7080: 'varicose veins',
+  7090: 'vasectomy',
+  7100: 'Vincent\'s stomatitis',
+  7110: 'vision',
+  7120: 'visually impaired',
+  7130: 'vitamin deficiency',
+  7140: 'voice loss',
+  7150: 'vulvovaginitis',
+  7160: 'war neurosis',
+  7170: 'wasting syndrome',
+  7180: 'worms',
+  7190: 'wound, bullet',
+  7200: 'wound, shell fragment',
+  7210: 'wrist condition, bilateral',
+  7220: 'wrist condition, left',
+  7230: 'wrist condition, right',
+  7240: 'wrist sprain',
+  7250: 'wrist strain',
+  7260: 'yaws',
+  7270: 'yellow fever',
+  7280: 'hepatitis B',
+  7290: 'PTSD personal trauma',
+  7300: 'hepatitis C',
+  100: 'Addison\'s disease',
+  3350: 'hernia, ventral',
+  2065: 'diabetes, adult-onset',
+  2085: 'diabetes mellitus, type 2',
+  3035: 'Gulf War veteran with diagnosed illness as qualifying chronic disability',
+  530: 'ASHD',
+  8917: 'Miscellaneous',
+  8918: 'hip replacement',
+  8919: 'knee replacement',
+  109: '2010 Presumptive Herbicide Condition',
+  111: '2010 Presumptive Herbicide Condition',
+  8920: 'Adhesions - Digestive',
```

Comment:
From here down, the conditions are capitalized, which is inconsistent with the above. If the list is temporary, we can ignore it, but if it's ever going to be in production, we probably want to make them consistent.